app/routers/moderation.py

```
@@ -0,0 +1,142 @@
+        classification=classification,
+        confidence=confidence,
+        reasoning=reasoning,
+        status=req.status,
+    )
+
+
+@router.post("/moderate/image")
+async def moderate_image(
+    payload: ImageModerationRequest,
+    background_tasks: BackgroundTasks,
+    session: AsyncSession = Depends(get_session),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)